Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -3459,34 +3459,64 @@ def panorama_delete_url_filter_command(url_filter_name: str):
 
 
 def prettify_rule(rule: dict):
-    pretty_rule = {
-        'Name': rule['@name'],
-        'Action': rule['action']
+
+    parse_pan_os_un_committed_data(rule, ['@admin', '@dirtyId', '@time'])
+
+    rule_get = partial(dict_safe_get, rule, return_type=(str, list), raise_return_type=False)
+
+    # get rule devices:
+    entries = rule_get(['target', 'devices', 'entry'])
+    if not isinstance(entries, list):
+        entries = [entries]
+    rule_devices = [
+        entry.get('@name') for entry in entries if isinstance(entry, dict)
+    ]
+
+    pretty_rule: Dict[str, Any] = {
```

Comment:
lets call this context_rule as it isnt just for hr

Reply:
that was the original name and its used everwere, let me know if its critical